### PR TITLE
Grunt to Gulp

### PR DESCRIPTION
**Pull request by rhutchison**

### CODE REVIEW

**Comment by dotContent** [comment]
May want to rename this to "move from Grunt to Gulp" or something, haha.

**Comment by ilanbiala** [comment]
@lirantal are you good with switching to Gulp?

File: README.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ This command does a few things:
 * Finally, when the install process is over, npm will initiate a bower install command to install all the front-end modules needed for the application
 
 ## Running Your Application
-After the install process is over, you'll be able to run your application using Grunt, just run grunt default task:
+After the install process is over, you'll be able to run your application using Gulp, just run gulp default task:
```
Comment (ilanbiala) [inline README.md]: This could be cleaned up a bit

File: server.js [hunk 2]
```diff
@@ -6,7 +6,9 @@
 var config = require('./config/config'),
 	mongoose = require('./config/lib/mongoose'),
 	express = require('./config/lib/express'),
-	chalk = require('chalk');
+	chalk = require('chalk'),
+	q = require('q'),
```
Comment (ilanbiala) [inline server.js]: From a strict performance perspective, bluebird is a better library than q, no?

**Comment by ilanbiala** [comment]
Shouldn't the package.json change be handled already because of the recent merge?

**Pull request closed without merging**